addons/website/static/src/js/editor/snippets.editor.js

```diff
@@ -38,6 +38,12 @@ weSnippetEditor.SnippetsMenu.include({
     async start() {
         await this._super(...arguments);
         this.$currentAnimatedText = $();
+        // TODO: Remove me in master since it will be patched by an XML change
+        // grep: KEA_BG_SHAPES
+        const $faultyShapes = this.$body.find(`#wrapwrap [data-oe-shape-data='{"shape":"web_editor/Origins/Wavy_03","flip":["y"]}']`);
+        if ($faultyShapes.length) {
+            $faultyShapes.attr('data-oe-shape-data', `{"shape":"web_editor/Wavy/03","flip":["y"]}`);
+        }
```

Comment:
@qsm-odoo 
Noticed something odd with this line of code:
https://github.com/odoo/odoo/blob/6b12c64e09c92fba96532658f959daf7d234223c/addons/web_editor/static/src/js/editor/snippets.options.js#L6516

This seems to me like nothing is returned if the last shape is selected on the sibling.

Sure enough, in 14.0 (when these options were introduced) toggling the shape on an element dropped under a block that has the last shape selected results in nothing happening. Essentially preventing you to select a shape.

I opened a different PR in draft for 14.0 [here](https://github.com/odoo/odoo/pull/107951) since this pr is linked to the one in design-themes with the faulty template in 15.0. But the 14.0 fix will remove the traceback seen in the OPW, so it might be worth doing here.

Not sure what do to in these circumstances. What do you think?

Reply:
> But the 14.0 fix will remove the traceback seen in the OPW

Then we'll merge it, let it forward-port then you'll rebase here afterwards. Just mention the OPW in both commits :+1: